src/ECER.Resources.Documents/Applications/IApplicationRepository.cs

```diff
@@ -23,6 +23,20 @@ public record Application(string? Id, string ApplicantId, IEnumerable<Certificat
   public DateTime? SignedDate { get; set; }
   public DateTime? SubmittedOn { get; set; }
   public PortalStage Stage { get; set; }
+  public IEnumerable<Transcript> Transcripts { get; set; } = Array.Empty<Transcript>();
+}
+
+public record Transcript()
+{
+  public string? Id { get; set; }
+  public string EducationalInstitutionName { get; set; } = string.Empty;
+  public string ProgramName { get; set; } = string.Empty;
```

Comment:
setting default value for a mandatory field is probably not a good idea. If it is mandatory, we should use record constructor to enforce the population of these properties

Reply:
do we need validation in the repository as well? I thought we just need to do that in API